backend/src/club/club.service.ts

```diff
@@ -51,6 +52,17 @@ export class ClubService {
         return (await this.clubRepo.find()).map(club => new ClubViewModel(club));
     }
 
+    async checkIfAlreadyCheckedIn(juniorId: string, clubId: string): Promise<boolean> {
+        const checkIns = await this.getCheckinsForClub(clubId);
+        let duplicateCheckIn = false;
+        await checkIns.forEach((checkIn) => {
```

Comment:
Does this need await?

Reply:
It was used previously with await, so I went with it.